.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: Bug Needs Triage
```

Comment:
```suggestion
labels: ["Bug", "Needs Triage"]
```

https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms

According to this it should be delimited.